tests/physics/helicity_formalism/angular_distributions/conftest.py

```diff
@@ -0,0 +1,255 @@
+# cspell:ignore scipy, asarray, nquad, yerr, ylim, ylabel, histogramdd
+
+from functools import reduce
+from math import cos, sqrt
+from typing import Any, Callable, Dict, Optional, Sequence, Tuple
+
+import numpy as np
+import pytest
+import scipy.integrate as integrate
+from expertsystem.io import load_amplitude_model
+from matplotlib import pyplot as plt
+
+from tensorwaves.data.generate import generate_data
+from tensorwaves.physics.helicity_formalism.amplitude import IntensityBuilder
+from tensorwaves.physics.helicity_formalism.kinematics import (
+    HelicityKinematics,
+)
+
+
+class Histogram:
+    def __init__(
+        self,
+        var_name: str,
+        bin_edges: Sequence[Sequence[float]],
+        bin_contents: Sequence[float],
+        bin_errors: Sequence[float],
+        **mpl_kwargs: Any,
+    ) -> None:
+        self.var_name = var_name
+        self.bin_edges = bin_edges
+        self.bin_contents = bin_contents
+        self.bin_errors = bin_errors
+        self.mpl_kwargs = mpl_kwargs
+
+
+def __chisquare(
+    values: Sequence[float], errors: Sequence[float], expected: Sequence[float]
+) -> float:
+    return np.sum(
+        [((x[0] - x[1]) / x[2]) ** 2 for x in zip(values, expected, errors)]
+    )
+
+
+@pytest.fixture(scope="module")
+def chisquare_test() -> Callable[[Histogram, Callable], None]:
```

Comment:
I wonder if it isn't more transparent and convenient to merge the `angular_distributions` folder into one `test_angular_distributions.py` test file that contains the function definitions that have to be defined as fixtures under `conftest.py`.